src/vs/platform/terminal/common/terminal.ts

```diff
@@ -49,6 +49,7 @@ export const enum TerminalSettingId {
 	MacOptionClickForcesSelection = 'terminal.integrated.macOptionClickForcesSelection',
 	AltClickMovesCursor = 'terminal.integrated.altClickMovesCursor',
 	CopyOnSelection = 'terminal.integrated.copyOnSelection',
+	MultiLinePasteWarning = 'terminal.integrated.multiLinePasteWarning',
```

Comment:
Let's change the setting to `enableMultiLinePasteWarning` to align closer with most other boolean settings